cot/src/response/into_response.rs

```diff
@@ -1,9 +1,13 @@
 use bytes::{Bytes, BytesMut};
 use cot::headers::{HTML_CONTENT_TYPE, OCTET_STREAM_CONTENT_TYPE, PLAIN_TEXT_CONTENT_TYPE};
+use cot::json::Json;
 use cot::response::Response;
-use cot::{Body, StatusCode};
+use cot::{Body, Error, StatusCode};
 use http;
+use serde::Serialize;
```

Comment:
I would recommend removing `use cot::json::Json` and `serde::Serialize` as they both are only used if `json` feature is enabled, and using fully-qualified paths later on in the file instead. Adding one `#[cfg(feature = "json")]` is fine for one `use`, but more than one and it's becoming a mess.